activemq/manifest.json

```diff
@@ -24,5 +24,12 @@
     "windows"
   ],
   "type": "check",
-  "integration_id": "activemq"
+  "integration_id": "activemq",
+  "assets": {
```

Comment:
Should this be called `assets` or `resources`?

Reply:
based on the internal talk today, we'll go with `assets`

Reply:
I'm closing this in favor of - https://github.com/DataDog/integrations-core/pull/3768